### PR TITLE
article: Donut

### PR DESCRIPTION
**Pull request by bescka**
* Added modelling article.

### CODE REVIEW

File: users/bescka/fn/cube_grid.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+  window.stopAnimation = function() {
+    if (tmr !== undefined) {
+      clearInterval(tmr);
+      tmr = undefined;
+    }
+  };
+
+
+  var startButton = document.getElementById('startCubeGridButton');
```
Comment (RGHANILOO) [inline users/bescka/fn/cube_grid.js:69]: both star and stop button variable querying the same ID

File: users/bescka/index.html [hunk 2]
```diff
@@ -60,7 +60,10 @@
   <link rel="icon" href="/assets/fc_favicon_96_2.ico" type="image/x-icon">
   <link rel="shortcut icon" href="/assets/fc_favicon_96_2.ico" type="image/x-icon">
   <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap" rel="stylesheet">
-  <script src="script.js"></script>
+  <script src="fn/script.js"></script>
```
Comment (RGHANILOO) [inline users/bescka/index.html:63]: this is absolutely fine as is, but might worth looking into [import/export](https://javascript.info/import-export) and bring in a single script

File: users/bescka/index.html
Comment (RGHANILOO) [inline users/bescka/index.html:1]: - lines 1-50 can maybe extracted to readme.md or even in the about section of the website

- there is currently a menu button on the page used as navigation, as a user when i press the button and navigate to any of the given option i have to press anywhere outside of it to close those options 
- navigation could also be more [semantic](https://developer.mozilla.org/en-US/docs/Glossary/Semantics#semantic_elements)

**Review by RGHANILOO — APPROVED** [review]
i am loving how this blog site is coming along. Nice work, this awesome.
🚀

File: users/bescka/fn/cube_grid.js (hunk 1 above)
Comment (bescka) [reply users/bescka/fn/cube_grid.js:69]: Thanks! Great spot :) Changed

File: users/bescka/index.html (hunk 2 above)
Comment (bescka) [reply users/bescka/index.html]: Thanks! Was a bit of a mess - I've refactored the script module logic/ split out and imported in index.html. Much nicer to read! I'll do the same for the other script elements when I get a chance.

File: users/bescka/index.html
Comment (bescka) [reply users/bescka/index.html:1]: 1) Good point, I've taken it out for now. Thought might be cool to have an easter-egg devlog on inspecting the page to explain stuff like the content loading but was a bit cluttered. 
2) Thanks - added an event listener to mouseleave in dropdown.js.
3) I've put in explicit paths in the anchor elements as it was a bit confusing. 

Also ended up putting a fix in for that behaviour where the site would default to home on refresh in the content_loader file. Seems to work!

**Pull request merged**